### ISSUE TITLE
no  reduction in loss

### ISSUE THREAD

**Issue by amritbhaskar** [chasingbob/squeezenet-keras#3]
The loss during training is not reducing. loss is constant at around  0.69 and and accuracy  at around 0.5

**Comment by jkstyle2** [comment]
@amritbhaskar Hello, I have a same issue. Did you solve it?
